retrievalmarket/storage_retrieval_integration_test.go

```diff
@@ -61,7 +63,7 @@ func TestStorageRetrieval(t *testing.T) {
 			pricePerByte:            abi.NewTokenAmount(1000),
 			paymentInterval:         uint64(10000),
 			paymentIntervalIncrease: uint64(1000),
-			voucherAmts:             []abi.TokenAmount{abi.NewTokenAmount(10136000), abi.NewTokenAmount(19920000)},
+			voucherAmts:             []abi.TokenAmount{abi.NewTokenAmount(10174000), abi.NewTokenAmount(19958000)},
```

Comment:
still confused about the voucher changes

Reply:
👍  @dirkmc let's follow up on this!

Reply:
See above. Nothing fishy here.